bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,596 @@
+			<channel id="controlProgram" typeId="controlProgram">
+				<label>Set Program</label>
+			</channel>
+			<channel id="controlHeatCircProgram0" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 0</label>
+			</channel>
+			<channel id="controlHeatCircProgram1" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 1</label>
+			</channel>
+			<channel id="controlHeatCircProgram2" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 2</label>
+			</channel>
+			<channel id="controlHeatCircProgram3" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 3</label>
+			</channel>
+			<channel id="controlHeatCircProgram4" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 4</label>
+			</channel>
+			<channel id="controlHeatCircProgram5" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 5</label>
+			</channel>
+			<channel id="controlHeatCircProgram6" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 6</label>
+			</channel>
+			<channel id="controlHeatCircProgram7" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 7</label>
+			</channel>
+			<channel id="controlHeatCircProgram8" typeId="controlHeatCircProgram">
+				<label>Set Program of Heat Circle 8</label>
+			</channel>
+			<channel id="controlWwHeat0" typeId="controlWwHeat">
+				<label>Trigger Warm Wather Circle 0</label>
+			</channel>
+			<channel id="controlWwHeat1" typeId="controlWwHeat">
+				<label>Trigger Warm Wather Circle 1</label>
+			</channel>
+			<channel id="controlWwHeat2" typeId="controlWwHeat">
+				<label>Trigger Warm Wather Circle 2</label>
+			</channel>
+			<channel id="controlExtraWwHeat0" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 0</label>
+			</channel>
+			<channel id="controlExtraWwHeat1" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 1</label>
+			</channel>
+			<channel id="controlExtraWwHeat2" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 2</label>
+			</channel>
+		</channels>
+		<properties>
+			<property name="boilerApproval">K0010</property>
+			<property name="program">PR001</property>
+			<property name="heatCircProgram">HKx01</property>
+			<property name="wwHeat">BKx06</property>
+			<property name="extraWwHeat">ZKx06</property>
+		</properties>
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the Guntamatic Heating System</description>
+			</parameter>
+			<parameter name="key" type="text" required="false">
+				<context>key</context>
+				<label>Key</label>
+				<description>Optional, but required to read protected parameters and to control the Guntamatic Heating System.
+					The
+					key needs to be reqested from Guntamatic support.</description>
```

Comment:
Looks like there is a carriage return you can suppress.
Same comment for all other thing types.

Reply:
👍

```
				<description>Optional, but required to read protected parameters and to control the Guntamatic Heating System.
					The key needs to be reqested from Guntamatic support.</description>
```

Reply:
I see no change, did you really change it ?
Is it when you apply spotless that this strange formatting is applied ?

Reply:
Yes, spotless:apply is always coming back with the strange format :-(

Reply:
Ok so we can forget this comment.